maps/map_files/USS_Runtime/USS_Runtime.dmm

```diff
@@ -92,7 +92,7 @@
 	},
 /area/event)
 "z" = (
-/obj/effect/landmark/start/pilot,
+/obj/effect/landmark/start/pilot/dropship_pilot,
```

Comment:
So we have a dropship_pilot landmark, but we ought to have the cas_pilot one too.